docs/plugins/google-tagmanager/client.js

```diff
@@ -0,0 +1,19 @@
+import ExecutionEnvironment from '@docusaurus/ExecutionEnvironment';
+
+export default (function () {
+  if (!ExecutionEnvironment.canUseDOM) {
+    return null;
+  }
+
+  return {
+    onRouteUpdate({location}) {
+      if (location) {
+        setTimeout(() => {
+          window.dataLayer.push({
+            event: 'route-update',
+          });
+        }, 50);
```

Comment:
I don't think so. It won't be used anywhere else... what would be the reason?

Reply:
if we want to change it it's a bit easier, but that's minor

Reply:
If it was config based, it would have to import from a separate config package, since this precise event name will be used on other sites as well, and must correspond to the configured event name in GTM

Reply:
oh I was talking about the timeout value. but anyway, it's minor

Reply:
OH I see, yeah that's also unlikely to change, I've copied it from Gatsby's official gtm plugin, so it's their number based on experience, probably making sure the route update process is really complete, i.e. `react-helmet` has had a chance to do its thing etc